scripts/send_reminder_notification.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * This script looks for todos with open reminders and reminds the assignees
+ */
+namespace {
```

Comment:
I just tried to match the other scripts in the folder. Dont mind adding the namespace